releasenotes/notes/add-litinski-transformation-ec42e98cc378613f.yaml

```
@@ -0,0 +1,40 @@
+---
+features_transpiler:
+  - |
+    Added a new transpiler pass :class:`qiskit.transpiler.passes.LitinskiTransformation`
```

We don't need to display the full path 🙂 
```suggestion
    Added a new transpiler pass :class:`~qiskit.transpiler.passes.LitinskiTransformation`
```

---

Done in [bc76da4](https://github.com/Qiskit/qiskit/pull/14753/commits/bc76da40133040bc8ec01fd8d315dac8a030bdce).